python/cusignal/_upfirdn.py

```diff
@@ -229,16 +190,8 @@ def _raw_apply_1d(tpb, bpg, x, h_trans_flip, out, up, down, axis=-1):
 
     _init_raw_apply1d_modules()
     kernel_double = _cached_modules['_raw_apply_1d_double']
-    kernel_int = _cached_modules['_raw_apply_1d_int']
 
-    if out.dtype == cp.int:
-        kernel_int(
-            (bpg,),
-            (tpb,),
-            (n, x_shape_a, h_per_phase, padded_len,
-             up, down, xx, xh_trans_flip, xout,),
-        )
-    elif out.dtype == cp.float64:
+    if out.dtype == cp.float64:
```

Comment:
what do you do if the out type is not float64?

Reply:
@BradReesWork In my update to this code, I've added float32, complex64, and complex128 to cover all scenarios.

Reply:
@mnicely -- Are you referring to https://github.com/rapidsai/cusignal/pull/31? Is this ready for review?

Reply:
Yes. I copied _upfirdn.py from this PR to use as my baseline because it had your latest changes.